ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -169,6 +176,10 @@ class NavigationControllerDemoController: DemoController {
         if showAvatar {
             controller.msfNavigationBar.personaData = content.personaData
             controller.msfNavigationBar.onAvatarTapped = handleAvatarTapped
+        } else if showStarButton {
```

Comment:
Are `showAvatar` and `showStarButton` supposed to be exclusive?

Reply:
Yes, this is by design. As per `LargeTitleView`'s [`updateProfileButtonVisibility`](https://github.com/microsoft/fluentui-apple/blob/main/ios/FluentUI/Navigation/Views/LargeTitleView.swift#L310-L312) method, we don't support showing a custom leading button and an avatar at the same time. We can show a back button with an avatar at the same time, but this is only because the back button is REALLY important. Clients have explicitly requested the "avatar + back button + two-line title" case, and I think that looks okay, but swapping in a single large title looks somewhat gross in my opinion.

| Back button and avatar, large title | Back button and avatar, two-line title |
|-|-|
| ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-15 at 15 37 18](https://user-images.githubusercontent.com/717674/219218513-5f6397e7-733b-4c57-9702-8550aed35d89.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-15 at 15 37 22](https://user-images.githubusercontent.com/717674/219218557-c3e4b4b3-c6df-438b-9848-c4e14e61c70b.png) |

Reply:
That being said, I think we can clean up `presentController` to make this a bit more clear.